insights/parsers/tests/test_du_usage.py

```diff
@@ -0,0 +1,62 @@
+from insights.parsers.du_usage import DiskUsage
+from insights.tests import context_wrap
+
+du_content = """
+22G     /var/lib/mongodb
+4.0K	/var/lib/pulse
+113M	/var/lib/rpm
+4.0K	/var/lib/samba/private
+8.0K	/var/lib/samba
+8.0K	/var/lib/xkb
+4.0K	/var/lib/gssproxy/clients
+4.0K	/var/lib/gssproxy/rcache
+12K	/var/lib/gssproxy
+4.0K	/var/lib/net-snmp/cert_indexes
+4.0K	/var/lib/net-snmp/mib_indexes
+12K	/var/lib/net-snmp
+56K	/var/lib/upower
+4	/root
+92	/opt/google/chrome/default_apps
```

Comment:
This looks like you've mixed 'du -h' and 'du' output here.  Are you sure this is accurate?

Reply:
And this comment pointed out by Paul should be fixed too.

Reply:
As per your current logic, there is not `du -h` at all. Test `du -sh` only is enough.

Reply:
yup, my bad that was mixed output, I have changed it.